src/client/rspamc.cxx

```diff
@@ -823,6 +825,59 @@ add_options(GQueue *opts)
 	}
 }
 
+static void
+print_indented_line(FILE *out, std::string line, size_t maxlen, size_t indent)
+{
+	if (maxlen < 1) return;
+
+	std::string s;
+	for (size_t pos = 0; pos < line.length(); pos += s.length()) {
+		s = line.substr(pos, pos ? (maxlen-indent) : maxlen);
+		if (indent && pos) fmt::print(out, "{:>{}}", " ", indent);
```

Comment:
```suggestion
	for (size_t pos = 0; pos < line.length(); pos += s.length()) {
		auto s = std::string_view(line).substr(pos, pos ? (maxlen-indent) : maxlen);
		if (indent && pos) {
			fmt::print(out, "{:>{}}", " ", indent);
		}
```

Reply:
Can't define "s" as "auto" inside the "for" loop as otherwise "s.length()" becomes outside the scope and compiler throws the error.